modules/overview-of-component-resource-customization.adoc

```diff
@@ -0,0 +1,43 @@
+:_module-type: CONCEPT
+
+[id='overview-of-component-resource-customization_{context}']
+= Overview of component resource customization
+
+[role='_abstract']
+You can customize deployment resources that are related to the {productname-long} Operator, for example, CPU and memory limits and requests. For resource customizations to persist without being overwritten by the Operator, the `opendatahub.io/managed: true` annotation must not be present in the YAML file for the component deployment. This annotation is absent by default.
```

Comment:
"CPU and memory limits and requests." Could this be reworded?

Reply:
Discussed offline, leaving as-is.